You're right. The 2 tests in the JakartaSecurityTest class fail, because of:

```
Caused by: org.osgi.framework.BundleException: Unable to resolve org.omnifaces.jwt [32](R 32.0): missing requirement [org.omnifaces.jwt [32](R 32.0)] osgi.wiring.package; (&(osgi.wiring.package=com.nimbusds.jose)(version>=9.23.0)(!(version>=10.0.0))) Unresolved requirements: [[org.omnifaces.jwt [32](R 32.0)] osgi.wiring.package; (&(osgi.wiring.package=com.nimbusds.jose)(version>=9.23.0)(!(version>=10.0.0)))]
```

--- a/appserver/pom.xml
+++ b/appserver/pom.xml
@@ -99,10 +99,10 @@
         <jakarta.interceptor-api.version>2.1.0</jakarta.interceptor-api.version>
 
         <!-- Jakarta Security + Authentication/Authorization -->
-        <soteria.version>3.0.3</soteria.version>
+        <soteria.version>3.0.4</soteria.version>
         <exousia.version>2.1.3</exousia.version>
         <epicyro.version>3.0.0</epicyro.version>
-        <nimbus.version>10.0.2</nimbus.version>
+        <nimbus.version>10.4.2</nimbus.version>
         <jcip.version>1.0.2</jcip.version>
 
         <!-- Jakarta Messaging -->
@@ -159,7 +159,7 @@
 
         <!-- MicroProfile JWT / JWT Authentication Mechanism for Jakarta Security -->
         <microprofile-jwt-auth-api.version>2.1</microprofile-jwt-auth-api.version>
-        <omnifaces-jwt-auth.version>2.0.2</omnifaces-jwt-auth.version>
+        <omnifaces-jwt-auth.version>2.0.3</omnifaces-jwt-auth.version>
 
         <!-- MicroProfile REST Client -->
         <microprofile.rest-client.version>3.0.1</microprofile.rest-client.version>

--- a/appserver/tests/application/pom.xml
+++ b/appserver/tests/application/pom.xml
@@ -39,7 +39,7 @@
             <groupId>org.glassfish.main.distributions</groupId>
             <artifactId>glassfish</artifactId>
             <type>zip</type>
-            <version>${project.version}</version>
+            <version>${glassfish.version}</version>
             <scope>test</scope>
         </dependency>
         <dependency>

